lib/logstorage/filter_and_test.go

```diff
@@ -114,4 +115,61 @@ func TestFilterAnd(t *testing.T) {
 		},
 	}
 	testFilterMatchForColumns(t, columns, fa, "foo", nil)
+
+	fa = &filterAnd{
+		filters: []filter{
+			&filterPrefix{
+				fieldName: "foo",
+				prefix:    "a foo",
+			},
+			&filterOr{
+				filters: []filter{
+					&filterExact{
+						fieldName: "foo",
+						value:     "a foobar",
+					},
+					&filterExact{
+						fieldName: "boo",
+						value:     "bbbbbbb",
+					},
+				},
+			},
+		},
+	}
+	testFilterMatchForColumns(t, columns, fa, "foo", []int{1})
+}
+
+func TestGetFilterTokens(t *testing.T) {
+	// check filterAnd and filterOr
+	fa := &filterAnd{
+		filters: []filter{
+			&filterExact{
+				fieldName: "foo",
+				value:     "baz",
+			},
+			&filterOr{
+				filters: []filter{
+					&filterExact{
+						fieldName: "foo",
+						value:     "a",
+					},
+					&filterExact{
+						fieldName: "foo",
+						value:     "b",
+					},
+				},
+			},
+		},
+	}
+	andToken := fa.getByFieldTokens()
+	if !reflect.DeepEqual([]fieldTokens{
+		{
+			field: "foo",
+			tokens: []string{
+				"baz",
+			},
+		},
+	}, andToken) {
+		t.Fatalf("unexpected tokens %v", andToken)
+	}
```

Comment:
It seems to me like this test is excessive after adding a test case which checks the same code paths above.

Reply:
thanks for reviewing, i delete this func.